front-end/src/util/parseBalance.ts

```diff
@@ -0,0 +1,5 @@
+export default (balance: string) => {
+	const result = parseInt(balance, 10);
+
+	return isNaN(result) ? 0 : Math.floor(result/1000000000000);
```

Comment:
This is not a good way to do KSM balance. is there any polkadot util function i can use

Reply:
yup there's a formatBalance ([used here(https://github.com/polkadot-js/apps/blob/7423ca9984cfc7cc3d44b0ae47c3d62b57db5321/packages/react-query/src/FormatBalance.tsx#L26)] for instance) but I'd just do what we do in the GovernanceInfo using the info we store in the chainProperties. [here](https://github.com/paritytech/polkassembly/blob/master/front-end/src/components/Post/PostProposalInfo.tsx#L38)

Reply:
Using format balance. We cannot query user account balance from chainProperties. It needs to be queried at realtime.